### ISSUE TITLE
Default slot content is not rendered when content of component is empty

### ISSUE THREAD

**Issue by valichek** [aurelia/templating#388]
Custom component:
```html
<template>

  <slot name="first">
    <p>First slot default</p>
  </slot>
  
  <slot name="second">
    <p>Second slot default</p>
  </slot>
  
</template>

```

Examples:

```html
<template>
  <require from="./custom"></require>
  
  <h2>Example 1: (rendered)</h2>
  <custom>
    <div slot="first">
      <p>First slot</p>
    </div>
  </custom>
  
  
  <h2>Example 2: (bug here)</h2>
  <custom></custom>
  
  <h2>Example 3: (rendered)</h2>
  <custom>
    <!--comment here-->
  </custom>
  
</template>

```

Here is Gist example

https://gist.run/?id=41b17ba5793844b1327aba09f3141352

**Comment by AStoker** [comment]
+1

**Comment by EisenbergEffect** [comment]
@valichek We've fixed the exception from the related issue. Is this particular issue still happening?

**Comment by valichek** [comment]
@EisenbergEffect The issue is still there, NOT fixed in `aurelia-templating@1.0.0-beta.3.0.5`

**Comment by EisenbergEffect** [comment]
Release is coming in just a minute.
